packages/core/src/components/document-card/document-card.scss

```diff
@@ -37,7 +37,8 @@
 ::slotted([slot="meta"]) {
   display: flex !important;
   justify-content: space-between !important;
-  margin-inline-start: auto !important;
+  flex-shrink: 0 !important;
+  align-self: end !important;
```

Comment:
Dit is code van Hans die ook op de plekInfoCard is toegepast

Reply:
Kleine aanpassing aan de code die we uitleveren ...